### DISCUSSION TITLE
Can you use MM firmware without a proper zstop and useing the crtouch for z offset?

### DISCUSSION THREAD

**Discussion by MaynardMann** [mriscoc/Ender3V2S1#370] [Q&A]
My bed is hopelessly warped and iv been trying to figure out a solution that isn't test printing dozens of squares and calibrating each point on the mesh by hand based on print results, I want to be able to manually probe the bed with the nozel like what is done with manual mesh but using the manual mesh firmware from special configs seems to completely disable my probe for z offset leaving my extruder hovering far above the bed. MM seems to be exactly what I want but it pears impossible with my hardware setup.

**Comment by mriscoc** [comment]
For use the manual mesh version you must connect the spare end stop in the Z limit interface of the board  

![image](https://user-images.githubusercontent.com/2745567/189806030-d6bd1558-a3e7-4eee-9abb-0efa66a5eede.png)
